lib/ai/tools.ts

```
@@ -30,19 +30,19 @@ const searchKnowledgeBase = async (query: string, mailbox: Mailbox) => {
   return documents ?? "No past conversations found";
 };
 
-const updateCustomerMetadata = async (email: string, mailboxId: number, mailboxSlug: string) => {
+const updateCustomerMetadata = async (email: string, unused_mailboxId: number, mailboxSlug: string) => {
```

ESLint

[error] 'mailboxSlug' is defined but never used. Allowed unused args must match /^_/u.

(@typescript-eslint/no-unused-vars)